ui/src/components/table/QTable.json

```
@@ -788,7 +788,17 @@
         "cols": {
           "type": "Object",
           "desc": "Column definitions",
-          "__exemption": [ "examples" ]
+          "__exemption": [ "examples" ],
+          "definition": {
+            "name": {
+              "type": "String",
+              "desc": "Column name"
+            },
+            "value": {
+              "type": "String",
+              "desc": "Column value for the given row"
```

The value is not limited to `String`, it can be anything. So, let's match the type and description with `body-cell`.
```suggestion
              "type": "Any",
              "desc": "Parsed/formatted value for the given row",
              "__exemption": [ "examples" ]
```